numpy/linalg/linalg.py

```diff
@@ -2431,6 +2432,9 @@ def norm(x, ord=None, axis=None, keepdims=False):
     The Frobenius norm is given by [1]_:
 
         :math:`||A||_F = [\\sum_{i,j} abs(a_{i,j})^2]^{1/2}`
+        
+    Both the Frobenius and nuclear norm orders are only defined for
+    matrices and raise a ValueError when `x.ndim != 2`.
```

Comment:
```suggestion
    matrices and raise a ValueError when ``x.ndim != 2``.
```
My fault - I originally suggested markdown syntax, not rST

Reply:
Yes I agree, fixed it just now!